### PR TITLE
Fix for most of DFL-3607: Minor styling issues in Network details

### PR DESCRIPTION
**Pull request by danfooo**

### CODE REVIEW

File: src/network/network_style.css [hunk 1]
```diff
@@ -359,6 +359,8 @@
 .network-detail-overlay .not-content
 {
   padding: 0;
+  font: menu;
+  font-size: 11px;
```
Comment (chriskr) [inline src/network/network_style.css]: The style for ui labels should ideally only be defined once. I suggest to either add your selector to the according declaration in ui.css (at the very top) or to add to the selectors there a more generic class name for ui labels.

**Comment by chriskr** [comment]
Other then that it looks good.

**Comment by danfooo** [comment]
Fixed it, but weird having two classnames on a const, but seemed like the smallest solution.

**Comment by hzr** [comment]
Looks good.

**Pull request merged**